ui/src/mixins/validate.js

```diff
@@ -34,6 +34,15 @@ export default {
       this.__validateIfNeeded()
     },
 
+    disable (val) {
+      if (val === true) {
+        this.resetValidation()
+      }
+      else {
+        this.__validateIfNeeded(true)
+      }
+    },
+
```

Comment:
When passing from "enable" to "disable" it makes sense to resetValidation().
But not the other way around. Let the submit trigger this, or whatever the developer has configured. So please remove the `this.__validateIfNeeded(true)`

Reply:
ok